stock_location_lockdown/models/stock_quant.py

```diff
@@ -16,8 +16,11 @@ def check_location_blocked(self):
             if record.location_id.block_stock_entrance:
                 raise ValidationError(
                     _(
-                        "The location %s is blocked and can "
-                        "not be used for moving the product %s"
+                        "The location %(location)s is blocked and can "
+                        "not be used for moving the product %(product)s"
```

Comment:
Don't you have to change the pot file?

Reply:
@BT-aestebanez Usually, the translation updates can be done after merge at https://translation.odoo-community.org/

Reply:
Thanks for the information :)